packages/vitest/src/node/reporters/basic.ts

```diff
@@ -0,0 +1,27 @@
+import { DefaultReporter } from './default'
+import type { UserConsoleLog } from '#types'
+
+export class BasicReporter extends DefaultReporter {
```

Comment:
Maybe it should just be extended from a BaseReporter? You don't need to override methods then.

Reply:
On the other side it makes the intend more clean and whenever something changes in `DefaultReporter` this change will go also into `BasicReporter`

Reply:
I don't think these two reporters share anything in common. Rendering of "simple" reporter is coded in "base" reporter. Default one just defines its own "live" reporting, and fallbacks to "base", when TTY is not supported.

Reply:
I would even add an explicit getter/property to toggle this behaviour instead of relying on "isTTY".